crm_claim_rma/wizards/claim_make_picking.py

```diff
@@ -147,6 +146,7 @@ def _get_picking_line_data(self, claim, picking, line):
             'location_id': self.claim_line_source_location_id.id,
             'location_dest_id': self.claim_line_dest_location_id.id,
             'note': self._get_picking_note(),
+            'origin_returned_move_id': line.origin_move.id,
```

Comment:
:warning:  This should only be done in case we're generating a return !

If we want to create a new delivery with the wizard, this original move will restrict the search of the `stock.quant` to be used for the delivery (in `stock.move` `action_assign`), as it will search for a quant having this move in its `history_ids`. However the only quant having this move in its history is on the external location whereas for an outgoing picking, we'll search for a quant in internal location.